templates/scenarios/js/non-sso-tab-default-bot/package.json.tpl

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "{%appName%}",
+    "version": "0.0.1",
+    "description": "",
+    "author": "",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "install:bot": "cd bot && npm install",
+        "install:tab": "cd tab && npm install",
+        "install": "npm run install:bot & npm run install:tab",
```

Comment:
Can consider to run in parallel.

Reply:
thought & means running in parallel? Please correct me if I was wrong :)

Reply:
`&` will run the commands in sequence, not in parallel. Can use some package like `concurrently`.